### PR TITLE
[#22] [Integrate] As a user, I can see the intro question when taking a survey

### LINKED ISSUE #22: [Integrate] As a user, I can see the intro question when taking a survey
## Why

App must display the intro question before taking a survey

## Acceptance Criteria

- Must display the intro question
- Must display question with `displayType` "intro"

## Resources

- [Design](https://www.figma.com/file/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=34%3A868)

### PR DESCRIPTION
**Pull request by ryan-conway**
- Closes https://github.com/nimblehq/flutter-ic-lydia-ryan/issues/22

## What happened 👀

- Created `AnswerIntroWidget`

## Insight 📝

- N/A

### CODE REVIEW

File: lib/screens/surveydetails/widget/answer_intro_widget.dart [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class AnswerIntroWidget extends StatefulWidget {
```
Comment (lydiasama) [inline lib/screens/surveydetails/widget/answer_intro_widget.dart]: I'm not sure if we use this widget in any place. 🤔  ~Or do you wanna refactor [these lines](https://github.com/nimblehq/flutter-ic-lydia-ryan/pull/88/files#diff-c2055110ded419d42cb58ddbd1a4ec723cc53b414f860d7a9a53fa3dd01e1958R148-R158)?~ -> It's header I misread. 😂 

BTW, where would we use this widget? 🙏🏻

Comment (ryan-conway) [reply lib/screens/surveydetails/widget/answer_intro_widget.dart]: It seems we already had the intro question working 🙈 I'll update it to refactor the intro into a separate widget like the other questions instead 😅

Comment (lydiasama) [reply lib/screens/surveydetails/widget/answer_intro_widget.dart]: @ryan-conway Go ahead. 😂

**Review by lydiasama — APPROVED** [review]

File: lib/screens/surveydetails/widget/answer_intro_widget.dart [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
```
Comment (markgravity) [inline lib/screens/surveydetails/widget/answer_intro_widget.dart]: What do you think about using `StatelessWidget` here? 🤔

Comment (ryan-conway) [reply lib/screens/surveydetails/widget/answer_intro_widget.dart]: Updated in [b127948](https://github.com/nimblehq/flutter-ic-lydia-ryan/pull/88/commits/b12794892c45840ff3c81c6bc1c979eb0853c0f8) 🙏🏻

File: lib/screens/surveydetails/widget/answer_intro_widget.dart [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          title,
+          style: Theme.of(context).textTheme.titleLarge,
```
Comment (manh-t) [inline lib/screens/surveydetails/widget/answer_intro_widget.dart:20]: 👍

**Review by manh-t — APPROVED** [review]

**Pull request merged**